src/main/kotlin/com/wgslfuzz/core/Resolver.kt

```diff
@@ -391,9 +391,29 @@ private fun resolveAstNode(
                         resolveTypeDecl(node.typeDecl, resolverState)
                     } ?: resolverState.resolvedEnvironment.typeOf(node.initializer!!),
                 )
+
+            // TODO: Error if the handle address space is used incorrectly
```

Comment:
I am a fan of going for "TODO(github issue url): Error if ..." so that we have issues to track these TODOs.

Reply:
I decided to fix this one because it is important for getting the correct reference type. I missed it before opening the PR.